src/main/java/jeresources/compatibility/minecraft/MobCompat.java

```diff
@@ -0,0 +1,87 @@
+package jeresources.compatibility.minecraft;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+
+import jeresources.api.conditionals.LightLevel;
+import jeresources.entry.MobEntry;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.MobEntity;
+import net.minecraft.entity.boss.dragon.EnderDragonEntity;
+import net.minecraft.entity.monster.BlazeEntity;
+import net.minecraft.entity.monster.GuardianEntity;
+import net.minecraft.entity.monster.MagmaCubeEntity;
+import net.minecraft.entity.monster.MonsterEntity;
+import net.minecraft.entity.monster.PatrollerEntity;
+import net.minecraft.entity.monster.PhantomEntity;
+import net.minecraft.entity.monster.PillagerEntity;
+import net.minecraft.entity.monster.SlimeEntity;
+import net.minecraft.entity.monster.VexEntity;
+import net.minecraft.entity.monster.WitchEntity;
+import net.minecraft.entity.passive.AnimalEntity;
+import net.minecraft.entity.passive.BatEntity;
+import net.minecraft.entity.passive.DolphinEntity;
+import net.minecraft.entity.passive.WaterMobEntity;
+import net.minecraft.world.World;
+import net.minecraftforge.registries.ForgeRegistries;
+
+public class MobCompat {
+    private static World world = MinecraftCompat.getWorld();
+    private static final HashMap<Class, Integer[]> MOB_XP = new HashMap<>();
+    private static final HashMap<Class, LightLevel> LIGHT_LEVEL = new HashMap<>();
+
+    public static void setLightLevel(MobEntry entry) {
+        Class entityClass = entry.getEntity().getClass();
+        entry.setLightLevel(LIGHT_LEVEL.get(entityClass));
+    }
+
+    public static void setExperience(MobEntry entry) {
+        Class entityClass = entry.getEntity().getClass();
+        System.out.println("The Entity Class: "+entityClass+" | The Mininum Exp: "+MOB_XP.get(entityClass)[0]);

```

Comment:
Either use the logger class or remove the system prints

Reply:
Whoops, forgot to remove this testing remnant. Sorry